st2client/st2client/shell.py

```diff
@@ -65,8 +64,7 @@ class Shell(BaseCLIApp):
     LOG = LOGGER
 
     SKIP_AUTH_CLASSES = [
-        TokenCreateCommand.__name__,
-        LoginCommand.__name__
```

Comment:
On a related note - we need a test case to catch this.

If `LoginCommand` class is not here, st2 login won't work so we should have a test which catches this :)